recipes/win10toast/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "win10toast" %}
+{% set version = "0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/jithurjacob/Windows-10-Toast-Notifications.git
```

Comment:
Please install from a tarball rather than a repo

Reply:
Sadly, there is no tarball available for this project (on PyPi or on GitHub releases).

Reply:
Github will generate a release for you if it is tagged though

Reply:
There are no tags on the repo either :/.

Reply:
Hmmm I think we need to encourage the devs to release an actual source tarball, either with `twine sdist` or by git tag

Reply:
Agree, but as I mentioned in a [comment](https://github.com/conda-forge/staged-recipes/pull/11051#issuecomment-599274337), it is a package that hasn't been updated in a while even for simple fixes (one of which required the patch), and has multiple open, unresponded PRs. The absence of this recipe was blocking a version update on [another project](https://github.com/conda-forge/knockknock-feedstock/pull/4) that I contribute to

Reply:
I agree with Anthony that the developers should be encouraged to release an official tarball, so opening an issue regarding wouldn't hurt. 

Looks like the PyPI wheel was updated on 01/25/2018 which corresponds with commit "9d52b73f1af6c60162cf09b99269c4f7b13cdb00", so maybe `url: https://github.com/jithurjacob/Windows-10-Toast-Notifications/archive/9d52b73.tar.gz` can be used in the meantime ?

Reply:
If there's no tag in github, I don't see an issue with using the wheel.

Reply:
Yeah, that's why I used a wheel at start. Happy to use the tarball from the latest commit and opening an issue at source requesting a tag.

Reply:
Switched source to the commit tarball, opened an [issue](https://github.com/jithurjacob/Windows-10-Toast-Notifications/issues/66) at source.